lib/terminus-bootstrap.service.spec.ts

```diff
@@ -89,6 +89,7 @@ describe('TerminusBootstrapService', () => {
           '/up': expect.any(Function),
         },
         logger: expect.any(Function),
+        signal: 'SIG_NOT_EXIST',
```

Comment:
would it be better to move this "magic strings" to a dedicated `constant`?

Reply:
Done